p4rt_client/p4rt_client.go

```diff
@@ -62,6 +87,8 @@ type P4RTClientParameters struct {
 	Name       string
 	ServerIP   string
 	ServerPort int
+	Username   string
+	Password   string
```

Comment:
Can't figure out where are these two field set/used. Maybe you meant to set them in InitfromJson?

Reply:
My intention was to have these in the JSON file too, but then i changed my mind without deleting these. Removed.